CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contributing
+
+Contributing is welcomed! We have extensive documentation below as well as comments in the code to help you out.
+Please take a look at our GitHub issues for ideas on what to work on. Our Github issues are labeled by priorities. Critical being the highest, and low being the lowest. (Brownie points if you finish something from our [Initial Release Features Milestone](https://github.com/Heptagram-Bot/api/milestone/1).) We have a variety of tags you can sort issues by to find the right one for you. You cann sort by [Good First Issue](https://github.com/Heptagram-Bot/api/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22), [Help Wanted](https://github.com/Heptagram-Bot/api/issues?q=is%3Aopen+is%3Aissue+label%3A%22%F0%9F%91%90+help+wanted+%F0%9F%91%90%22), and more are being added all the time. Please make sure to check for issues flaged with things like "Staff Only" or "ticket work required". These are issues that need staff attention, and should not be assigned to people. Also keep you eye out for things marked as "assigned to user".
+
+## Workflow
+
+### Step 1: Find an issue
+
+Step 1 is to find an issue [Issues Page](https://github.com/Heptagram-Bot/api/issues) that you can do. If you comment .take in the issue, it will be assigned to you.
+
+### Step 2: Create a fork or branch
+
+All work should be done on a fork unless you have been added as a contributor. In that case, please use a branch off of master. We update our branches with changes frequently, so make sure to [keep your fork up to date](https://dev.to/giannellitech/keeping-your-fork-up-to-date-klh)
```

Comment:
IMHO, once we're at v0.8.0+, it would be best if everyone including me and the contributors don't directly touch the `master` branch...

It should be Create a fork and a branch.